src/components/pacbio/PacbioRunInfoEdit.vue

```diff
@@ -16,6 +16,7 @@
       </traction-field-group>
 
       <traction-field-group
+        v-if="!smrtLinkVersionv12SequelIIe"
```

Comment:
I believe both v12_revio and v12_sequel_iie don't need `DNA Control Complex Box Barcode` so this could be changed to check both.